sonolark/cmd/root.go

```diff
@@ -62,7 +63,7 @@ func getRootCmd(env map[string]string) *cobra.Command {
 	root := &cobra.Command{
 		Use:   "sonolark",
 		Short: "Sonolark is a tool which allows users to easily build scripts using the Starlark language on top of our library of useful functions including assertions, Kubernetes API access, and more.",
-		RunE: func(cmd *cobra.Command, args []string) error {
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Rather than changing this to `Run` and handling each case, I would recommend leaving it as `RunE` and updating the method that executes this.

That way you only have to handle all the errors in one place.

Currently we have:

```
	err := getRootCmd(getEnvs()).Execute()
	if err != nil {
		os.Exit(1)
	}
```
and you'd just need to add the printing.

Otherwise, I think an unintended consequence of this is that you're going to be reporting an error to the user but the exit code of the process is probably always 0 since an error is never returned. So yes, you're hiding the usage, but because you've told it you never encounter an error.

The exit code is something not particularly documented or tested, but users will expect a non-zero exit code on failure. K8s will even recognize that as a failure and thereby trigger different states in the containers etc and people use that to script around as well.